vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/GridViewUsingRenderersPage.java

```diff
@@ -74,8 +106,51 @@ private void createBasicRenderers() {
             dataProvider.refreshAll();
         })).setWidth("100px").setFlexGrow(0);
 
+        Collection<Button> editButtons = Collections
+                .newSetFromMap(new WeakHashMap<>());
```

Comment:
Disabling the buttons doesn't seem relevant for the test setup.

Reply:
It isn't, but it avoids multiple rows to be edited at once. It was just copied from `GridViewEditorPage`. I can remove the disabling logic if you find it necessary.

Reply:
Let's remove everything that isn't necessary.

Reply:
@sosa-vaadin Doesn't look like this is resolved yet?